src/eom-properties-dialog.c

```diff
@@ -401,6 +408,23 @@ pd_close_button_clicked_cb (GtkButton *button,
 	eom_dialog_hide (EOM_DIALOG (user_data));
 }
 
+static void
+pd_folder_button_clicked_cb (GtkButton *button, gpointer data)
+{
+	EomPropertiesDialogPrivate *priv = EOM_PROPERTIES_DIALOG (data)->priv;
+	GdkScreen *screen;
+	guint32 timestamp;
+
+
+	if (!priv->folder_button_uri)
+		return;
+	
+	screen = gtk_widget_get_screen (GTK_WIDGET (button));
+	timestamp = gtk_get_current_event_time ();
+
+	gtk_show_uri (screen, priv->folder_button_uri, timestamp, NULL);
```

Comment:
This gives me a deprecated warning. We should use `gtk_show_uri_on_window` instead.

Reply:
I will fix that with fixing another gtk_show_uri waring in code. I have local prepared a back port of this one https://gitlab.gnome.org/GNOME/eog/commit/77f16df

Reply:
@vkareh 
`gtk_show_uri_on_window` needs a GtkWindow as parent, but eom-properties-dialog is a GtkDialog window.
In eog the dialog is ported to a GtkApplication window after porting away from GtkAction.
So the solution from them doesn't work.
https://gitlab.gnome.org/GNOME/eog/commit/ac4f44b56554459c2fee7b849635ce7769230cfc
Any Idea?